docs/2023/BeckN/Backn Smart Policy Infrastructure/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 sidebar_position: 1
 title: Introduction
-slug: /2023/BeckN/BeckN Policy Admin Api
```

Comment:
Why you removed the slug here?

Reply:
I think that is like an extra folder I have checkout PR and there are two duplicate folders. I think I will remove this folder.